pkg/utils/utils.go

```diff
@@ -16,3 +16,29 @@ func ParseSymbol(address uint64, table *helpers.KernelSymbolTable) *helpers.Kern
 	hookingFunction.Owner = strings.TrimSuffix(hookingFunction.Owner, "]")
 	return hookingFunction
 }
+
+func HasBit(n uint64, offset uint) bool {
+	return (n & (1 << offset)) > 0
+}
+
+func ClearBit(n *uint64, offset uint) {
+	*n &= ^(1 << offset)
+}
+
+func SetBit(n *uint64, offset uint) {
+	*n |= (1 << offset)
+}
+
+func Min(x, y uint64) uint64 {
```

Comment:
Especially this and `Max` could be useful as generic shorthands.

Reply:
ditto